EasyPost.Tests.Net/ClientTest.cs

```diff
@@ -43,7 +43,7 @@
 //            Request request = new Request("resource");
 
 //            List<String> parameters = ClientManager.Build().PrepareRequest(request).Parameters.Select(parameter => parameter.ToString()).ToList();
-//            CollectionAssert.Contains(parameters, "user_agent=EasyPost/v2 CSharp/" + ClientManager.Build().version);
+//            CollectionAssert.Contains(parameters, "user_agent=EasyPost.NetFramework/v2 CSharp/" + ClientManager.Build().version);
```

Comment:
Comment fix here

Reply:
Resolved in [d903ec7](https://github.com/EasyPost/easypost-csharp/pull/214/commits/d903ec7476be02f732e6eb4245a85595fc72df59)